README.rst

```diff
@@ -116,8 +116,11 @@ There are several things you can customize in OpenCensus:
   other exporters are provided as `extensions <#trace-exporter>`__.
 
 * **Sampler**, which determines how traces are sampled.
```

Comment:
One thing we probably need to consider - update the README file so first time users wouldn't be surprised when there is no trace dumped to the console due to the default low sample rate:

```
from opencensus.trace.tracer import Tracer
tracer = Tracer()
with tracer.span('foo'):
    pass
```

Reply:
Added in 014b906, it does make the docs a bit uglier though.